lib/indexer/popularity_lookup.rb

```diff
@@ -40,11 +40,13 @@ def lookup_popularities(links)
           popularity_score = 1.0 / (ranks[link] + SearchConfig.popularity_rank_offset)
         end
 
+        popularity_rank = traffic_index_size - (ranks[link] || traffic_index_size)
```

Comment:
See https://github.com/alphagov/search-api/pull/1731#discussion_r332426000

Reply:
This is fixed in that PR. Are you OK for this to be merged?